examples/doc.org

```diff
@@ -13,8 +13,7 @@
 # limitations under the License.
 
 #+TITLE: Example documentation
-#+LANGUAGE: en
-#+OPTIONS: author:nil date:nil
+#+SETUPFILE: examples/config.org
```

Comment:
This path is relative to the workspace root, but I think it's expecting a path relative to this file?

Reply:
The "it" in "it's expecting" wasn't Bazel, but org-mode. To give a concrete example, consider the documentation for magit, which is under `docs/magit.org` under the repo root:  
https://github.com/magit/magit/blob/main/docs/magit.org?plain=1#L12

That refers to `.orgconfig` via a relative path, but it's a relative path from the `.org` file with the directive, not from the repo root. Since the relative paths in those `#+SETUPFILE` directives can be relative to something other than the repo root.

(They aren't necessarily relative to the Blaze package the target is in, either, since you can have something like `srcs = ["docs/magit.org"]` in the example above.)

Reply:
As with other files in Emacs, these are relative to `default-directory`, whatever that might be. Here I guess either leaving `default-directory` pointing to the execroot or mimicking file visiting and setting it to the directory of the file being visited both make some sense. Both aren't ideal because generated files (in the `bin` or `genfiles` trees) won't be easily reachable either way, but fixing that would probably required more significant changes (multiple inclusion roots similar to `load-path` etc.). I've now switched to the second option (changing the default directory); I'm not 100% convinced it's clearly better, but at least has the potential to avoid necessary patches of external repositories.